src/index.js

```diff
@@ -44,3 +123,158 @@ function makeCustomerNameCheck() {
   }
   return customerOptions;
 }
+
+function addUserRoomBookings(id) {
+  let bookings = customer.getUserAllBookings(id);
+  let htmlToEnter = ''
+  bookings.forEach(booking => {
+    htmlToEnter += `
+    <section class="availability-section">
+      <div class="availability-room-num">
+      <p class="availability-room-p">Room ${booking.roomNumber}</p>
+      </div>
+      <div class="availability-room-info">
+      <p>Booking Date: ${booking.date}</p>
+      <p>Confirmation #: ${booking.id}</p>
+      </div>
+    </section>
+    `
+  });
+  $("#customer-bookings").html(`
+    ${htmlToEnter}
+    `)
+}
+
```

Comment:
Another way to do this is to use the jquery method .append(). It will not overwrite the cards that have been inserted before, and alway inserts from the bottom. Would save you maybe three lines of code, so probably not worth it but just fyi